project.yml

```diff
@@ -0,0 +1,77 @@
+name: IronRoadForChildren
+
+fileGroups:
+  - project.yml
+  - README.md
+
+options:
+  bundleIdPrefix: at.fhj
```

Comment:
Having a look on the old app I guess this will be `at.irfc` and the BundleIdentifier `at.irfc.app`. We should check this with the PO - but I guess we can still change this later :)